PlusContent/WFPSupport.md

```diff
@@ -0,0 +1,95 @@
+
+**Bottom Line:** Another firewall installed on a system does not conflict with Sandboxie Plus and can also be used to block programs (sandboxed or not), but its rules are typically global and based on program paths. The WFP implementation in Sandboxie Plus, on the other hand, offers the advantage of "per box" rules which affect only processes within a given sandbox (without specifying program paths).
+
+### Implementing Network Access Rules in Sandboxie Plus
+
+The Sandman UI provides us with a method for editing and testing network rules. Right-click on a box and select "Sandbox Options" from the drop-down menu (or simply double-click on a box) to bring up the Box Options UI. Then click on Network Options in the left panel and select the Network Firewall tab. The Test Rules row appears at the bottom, below the rule list (which may or may not be already populated). One can enter program name, port, IP address and protocol to see which rules are in play and which rule will be applied in the end. The choice of blocking (using WFP or by denying access to network devices) is selected in the Process Restrictions tab.
+
+![](../Media/WFP_Rule_Editor.png)
+
+The **attributes** at our disposal (with some examples of syntax) are:
+- **Action**=Allow|Block [Selected from the Network Restrictions tab]
+- **Program**=program.exe
+- **Port**=80,443,1000-2000
+- **IP**=111.222.333.444,0.0.0.0-255.255.255.255
+- **Protocol**=TCP|UDP 
+
+The following **rules precedence** scheme determines rule hierarchy:
+1. A rule for a specified program trumps a rule for all programs except a given one, 
+   trumps rules for all programs.
+2. A rule with a Port or IP trumps a rule without.  
+    2a. A rule with IP and Port trumps a rule with IP only or Port only.  
+    2b. A rule with one IP trumps a rule with an IP range that is besides that on 
+        the same level.
+3. Block rules trump Allow rules.
+4. A rule without a Protocol means all protocols.  
+   4a. A rule with a Protocol trumps a rule without, if it is the only difference.
+
+**Some Examples:**
+- NetworkAccess=Block;Port=80,443;IP=0.0.0.0-255.255.255.255 **[No Network Access]** <br> 
+NetworkAccess=Allow:Port=80,443;IP=111.222.333.444 **[Allow any program to access this IP]**
+
+- NetworkAccess=Block;Port=80,443;Protocol=TCP **[Block rule]** <br>
+NetworkAccess=Allow;Port=80,443;IP=111.222.333.444 **[Allow any program to access this IP]**
+
+- NetworkAccess=Block;Port=80,443 **[Block rule]** <br>
+NetworkAccess=chrome.exe,Allow;Port=80,443 **[Allow chrome.exe to access any IP]**
+
+- NetworkAccess=Block;Port=80,443 **[Block rule]** <br>
+NetworkAccess=chrome.exe,Allow;Port=80,443;IP=111.222.333.444 **[Allow chrome.exe to access one IP]**
+
+**Block Ports Rule:**
+- NetworkAccess=Block;Port=137,138,139,445 **[This replaces the old BlockPort template]**
```

Comment:
```suggestion
- NetworkAccess=*,Block;Port=137,138,139,445 **[This replaces the old BlockPort template]**
```

As mentioned in [Templates.ini#L3735](https://github.com/sandboxie-plus/Sandboxie/blob/19ab3030da9bee0df075ddf7e30e8135e5673f46/Sandboxie/install/Templates.ini#L3735) (it was updated a number of releases ago).